wrappingGsdk/fakegame/Program.cs

```diff
@@ -23,6 +18,11 @@ static void Main(string[] args)
                     Console.WriteLine($"Argument: {arg}");
                 } 
             }
+
+            HTTP_PORT = int.Parse(args[2]);
```

Comment:
does this work? HTTP_PORT is specified as const above. It shouldn't be a const now.

Reply:
Oh I didn't commit the latest version of file. sorry just updated it.